src/pages/index.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react'
 import { Link, graphql } from 'gatsby'
 import get from 'lodash/get'
-import Helmet from 'react-helmet'
 
 import Bio from '../components/Bio'
 import Layout from '../components/Layout'
+import SEO from '../components/SEO'
```

Comment:
This should be imported into _every_ component that will be served to the end user. It's currently just the index and blog post template, but additional pages are added (e.g. about, contact, etc.) definitely want to re-use this.